### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation (?)

### DISCUSSION THREAD

**Discussion by raravena80** [sustainable-computing-io/kepler#559] [General]
Hello Kepler team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I believe you had an agenda item to present the project in our meeting. I'm following up, I think it would be great for you to present/discuss the project in the future. I also think even though the focus of the project is cloud native sustainability there is overlap with TAG-Runtime.

Feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com) at your convenience. 

Thanks!

**Comment by rootfs** [comment]
@raravena80 sounds great! I added kepler intro to the Apr 6th meeting agenda. See you in a month!
